lib/src/test/java/vmath/Mat3Test.java

```diff
@@ -120,4 +120,45 @@ void givenArrayWithNaNs_whenMat3Created_thenThrowsException() {
         };
         assertThrows(ArithmeticException.class, () -> new Mat3(a2));
     }
+
+    @Test
+    void givenNoArgs_whenMat3Created_thenMatrixIsTransposed() {
+        var m = new Mat3();
```

Comment:
The code is good! I would recommend changing the test names to more accurately represent what they do, however: perhaps something like "givenIdentityMatrix_whenTransposed_thenResultIsCorrect()" and something similar with the second test. This will also match the naming style of other tests for previous functions of this sort.